### PR TITLE
chore(time-travel): add more options

### PR DESCRIPTION
**Pull request by haddeeann**
This is my first try at contributing to this project. And contributing to open source in general :)

I added some pages to the time travel dungeon area.

### CODE REVIEW

File: english/time-travel/0/5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+**You sit in the dark house alone and fall asleep. While you were sleep, a tiger enters your wood built house.**
```
Comment (hkumar0132) [inline english/time-travel/0/5.md:1]: sleep - sleeping
you can use Grammarly for grammar check

File: english/time-travel/0/4.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-**While you were sleep, a tiger enters your wood built house.**
-
-He kills you. 😐
+**It's getting dark, you have some extra fire wood leftover from building your house.**
```
Comment (hkumar0132) [inline english/time-travel/0/4.md:1]: fire wood -> firewood

File: english/time-travel/0/6.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+**You sit by the fire and enjoy the warmth**
+
+*You fall asleep by the fire and it's cozy. You awake in the morning in your new house. What do you do next?*
```
Comment (hkumar0132) [inline english/time-travel/0/6.md:3]: cozy -> cosy

File: english/time-travel/0/5.md (hunk 1 above)
Comment (haddeeann) [reply english/time-travel/0/5.md]: Thanks, I updated it. I don't know how to use Grammarly in markdown, so I will need to figure that out for the future.

File: english/time-travel/0/4.md (hunk 2 above)
Comment (haddeeann) [reply english/time-travel/0/4.md]: corrected

**Review by hkumar0132 — APPROVED** [review]
All set

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**